tests/sqllogictests/suites/tpch_iceberg/queries.test

```diff
@@ -7,7 +7,7 @@ TYPE=ICEBERG
 CONNECTION=(
     TYPE='rest'
     ADDRESS='http://127.0.0.1:8181'
-    WAREHOUSE='s3://iceberg-tpch'
+    `WAREHOUSE`='s3://iceberg-tpch'
```

Comment:
Now warehouse is a ident. So I need to modify this connection option.


BTW how about modify the key as warehouse_name?
https://github.com/databendlabs/databend/blob/ec9b07741bdeda3d5c3663d5959df4d0846afb7e/src/query/sql/src/planner/binder/ddl/catalog.rs#L221

Reply:
resolved.